Makefile

```diff
@@ -78,5 +78,8 @@ unit: generate testinsights
 e2e:
 	FOCUS="\[CustomerAdmin\]|\[EndUser\]\[Fake\]" TIMEOUT=60m ./hack/e2e.sh
 
+cvepatch:
```

Comment:
this will go away. it is here for now to allow me easily run the e2e tests for this particular piece of work. Triggering the job should be moved to the `release` repo